### DISCUSSION TITLE
मराठीत लिहिण्यासाठी मदत हवी आहे. - need help for Marathi typing

### DISCUSSION THREAD

**Discussion by sanketgarade** [mukta-strot/marathi-shabd#12] [Announcements]
मला मराठीत लिहिता येते, परंतु शुद्धलेखनाच्या बाबतीत बोंब आहे. त्यामुळे व्यवस्थित मराठी लिहिणारे लोक हवे आहेत.

प्रामुख्याने मराठीची आवश्यकता इथे आहे -

- डेटाबेस फाइलमध्ये मराठी शब्द जोडायला
- वेबसाईटचे इंग्रजीतून मराठीत भाषांतर करायला
- कागदपत्रांचे इंग्रजीतून मराठीत भाषांतर करायला

---

Currently most of the content is in English but I would like to have Marathi version for all documents and also help with getting Marathi words in the library database. 

So anyone who is good at and interested in creating stuff in Marathi is most welcome. 

Marathi will primarily be needed in -

- adding Marathi words to the library/database file
- The repo documentation 
- The website pages

**Comment by gokhaleaaroh** [comment]
नमस्कार, मला या प्रोजेक्टवर काम करायचे आहे, पण मला या प्रोजेकटचा नक्की गोल माहीत नाही. नक्की काय करणार आहोत आपण?  
मला python, java, C++ आणि  C# या भाषांमध्ये अनुभव आहे, तर मी काय मदत करू शकतो?

**Reply by sanketgarade** [reply]
नमस्कार. आपण readme वाचले आहे का? मी नुकतीच त्यात नवीन माहिती टाकली आहे. ध्येय (Goal) देखील नव्याने टाकले आहे. 
आपल्याला python येते तर आपण सध्या एक स्क्रिप्ट लिहू शकता का? db.csv फाईल मध्ये असलेला डेटा markdown फाईल मध्ये रुपांतर करायला? मी थोड्या वेळाने markdown चे टेम्प्लेट टाकेन. त्यावरून तुम्हाला कल्पना येईल. ती फाईल तय्यार केली की मी पुन्हा इथे सांगतो. धन्यवाद.

**Reply by gokhaleaaroh** [reply]
मी Issues मध्ये db.csv सॉर्ट करण्याबाबत वाचले, आणि एक python script तयार केली. ती आता pull request करतो.

**Reply by gokhaleaaroh** [reply]
pull request केली आहे

**Reply by sanketgarade** [reply]
Template is added. 
You can check it and also see issue #14

In the template there are 3 options right now. It's still WIP. But if you wish you can start with it. 

Also let's continue further discussion on the relevant issue or discussion posts.
